command/agent/job_endpoint.go

```diff
@@ -777,6 +777,7 @@ func ApiTgToStructsTG(job *structs.Job, taskGroup *api.TaskGroup, tg *structs.Ta
 	tg.Name = *taskGroup.Name
 	tg.Count = *taskGroup.Count
 	tg.Meta = taskGroup.Meta
+	tg.ShutdownDelay = *taskGroup.ShutdownDelay
```

Comment:
I guess the assumption here is that `taskGroup.ShutdownDelay` is never nil because of canonicalization?   The removal of the nil check makes me a tad nervous.